recipes/maddy/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.5)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(maddy REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} example.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE maddy::maddy)
+set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 14 CXX_STANDARD_REQUIRED
```

Comment:
Some validate logic is missing here :thinking:

Reply:
we don't have any validate logic in the CMakeLists.txt
we may add `validate` method to the recipe, if library requires C++14 to build.
but if test package requires C++14 standard, it's another story.